UrlManager.php

```diff
@@ -134,6 +134,18 @@ class UrlManager extends BaseUrlManager
      */
     public $languageParam = 'language';
 
+    /**
+     * @var string $_SERVER key set by apache mod_geoip
+     */
+    public $geoipServerFieldName = 'HTTP_X_GEO_COUNTRY';
+
+    /**
+     * @var array list of countries for given language code
```

Comment:
Better: "list of GeoIP countries indexed by corresponding language code". 

It should also have a note about the default value: "The default is an empty list which disables GeoIP detection".